tests/utils.py

```diff
@@ -316,48 +297,6 @@ def wait_for_resource_removal_in_db(
     return resource_removed
 
 
-def wait_for_resource_in_db(table, index, identifier, max_attempts=10, delay=0.5):
-    """Polls rethinkdb for the requested resource until it has been removed.
-    Useful when commiting a delete transaction in sawtooth and waiting for the
-    resource to be removed from rethink for dependent chained transactions.
-
-    Args:
-        table:
-            str:    the name of a table to query for the resource in.
-        index:
-            str:    the name of the index of the identifier to query for.
-        identifier:
-            str:    A id for a given resource to wait for.
-        max_attempts:
-            int:    The number of times to attempt to find the given role before
-                    giving up and returning False.
-                        Default value: 10
-        delay:
-            float:  The number of seconds to wait between query attempts.
-                        Default value: 0.5
-    Returns:
-        resource_removed:
-            bool:
-                True:   If the role is successfully found within the given
-                        number of attempts.
-            bool:
-                False:  If the role is not found after the given number of
-                        attempts.
-    """
-    resource_found = False
-    count = 0
-    with connect_to_db() as conn:
-        while not resource_found and count < max_attempts:
-            resource = (
-                r.table(table).filter({index: identifier}).coerce_to("array").run(conn)
-            )
-            if resource:
-                resource_found = True
-            count += 1
-            sleep(delay)
-    return resource_found
-
-
 def wait_for_role_in_db(role_id, max_attempts=10, delay=0.5):
```

Comment:
Was there a reason why some of these functions that queries the db are stil under tests/utils.py instead of db_queries.py?

Reply:
If I didn't touch it or use it I left, but I think I forgot to move it, because I do use that one.  I will move that thanks

Reply:
Ah okay - makes sense